doc/prelimsLinux.rst

```diff
@@ -0,0 +1,222 @@
+
+.. code-block:: shell
+
+   cd CMake-hdf5-1.8.19
+   ./build-unix.sh   # sudo might be required
+
+This will place the built binary in the bin folder and run through a series of tests for correct installation.
+
+d) Install and Compile mcopt Library
+************************************
+
+Finally, install the mcopt library itself; it can be found `here <https://github.com/jbradt/mcopt>`__. Clone the repository locally using the .git link found on GitHub.
+
+.. code-block:: shell
+
+   git clone https://github.com/jbradt/mcopt.git
+   cd mcopt
+
+The compilation and installation instructions can be found in the README.md file in this directory. The necessary commands are as follows.
+
+.. code-block:: shell
+
+   mkdir build && cd build
+   cmake -DCMAKE_BUILD_TYPE=Release ..
+   make
+   make install   # sudo might be required
+
+Test for correct code compilation by running the *test_mcopt* executable:
+
+.. code-block:: shell
+
+   ./test_mcopt
+
+4) Create a new Virtual Env
+---------------------------
+
+Now, create a virtual environment by entering the following into the command line:
+
+.. code-block:: shell
+
+   mkvirtualenv [name]
+
+Refer to the link in step 2 for information on using and managing virtual environments.
+
+
+
+5) Install the pytpc Package
+----------------------------
+
+Now, install the pytpc package and its dependencies; it can be found `here <https://github.com/ATTPC/pytpc.git>`__. Install the repository locally using the .git link found on GitHub.
+
+.. code-block:: shell
+
+   git clone https://github.com/ATTPC/pytpc.git
+   cd pytpc
+
+Installation instructions can be found in the README.md file. However, avoid Anaconda when for pytpc's purposes due to assorted problems with dependency versions and etc. Use pip to manage the required Python software packages.
+
+.. code-block:: shell
+
+   pip3.6 install Cython numpy scipy sklearn scikit-learn matplotlib seaborn jinja2 pandas clint pyYaml sqlalchemy tables h5py sphinx   # sudo might be required
+
+Then, to install pytpc from the source code, run:
+
+.. code-block:: shell
+
+   python3.6 setup.py install
+
+To test for correct installation. Run the provided tests with the following commands (not all tests print output to the screen but none should throw errors):
+
+.. code-block:: shell
+   
+   cd pytpc/tests
+   python3.6 test_evtdata.py
+   python3.6 test_gases.py
+   python3.6 test_grawdata.py
+   python3.6 test_hdfdata.py
+   python3.6 test_relativity.py
+   python3.6 test_simulation.py
+   python3.6 test_ukf.py
+   python3.6 test_utilities.py
+
+6) Create a Config File
+-----------------------
+
+Create a config file for the analysis code. There is a template in the next section of this documentation, or use the one created for argon-40 which can be found `here <https://github.com/jbradt/ar40-aug15/blob/master/fitters/config_e15503b.yml>`__.
+
+
+7) Set Up Energy Loss Data
+--------------------------
+Set up the energy loss info for the relevant nuclei.
```

Comment:
I actually automated that after writing the initial document you got from Daniel, so it should now happen automatically when you run `python setup.py install`. We'll still need to add a page to the docs about how to add a new particle / gas, though.